package.json

```diff
@@ -18,6 +18,7 @@
         "dayjs": "^1.11.13",
         "laravel-vite-plugin": "^1.0.5",
         "lodash": "^4.17.21",
+        "polish-plurals": "^1.1.0",
```

Comment:
Maybe use some more popular dependencie than `polish-plurals` ? 
For example https://vue-i18n.intlify.dev/guide/essentials/pluralization.html#pluralization

Or create you own

Reply:
done